### PR TITLE
update setup bot

### PR DESCRIPTION
**Pull request by IvanSakhman**

### CODE REVIEW

File: src/sample-account-setup.test.ts [hunk 1]
```diff
@@ -1,27 +1,546 @@
...
+          },
+        ],
+        photo: [
+          {
+            contentType: 'image/png',
+            url: 'https://docs.medplum.com/img/cdc-femaledoc.png',
+          },
+        ],
+      },
+      'Practitioner/practitioner'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:51]: I think this ID should be different, something like Practitioner/pid-12345

File: src/sample-account-setup.test.ts [hunk 2]
```diff
@@ -1,27 +1,546 @@
...
+        resourceType: 'Schedule',
+        id: 'schedule',
+        actor: [
+          {
+            reference: createReference(practitioner).reference,
+            display: getDisplayString(practitioner),
+          },
+        ],
+      },
+      'Schedule/schedule'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:69]: Same with this ID, something like Schedule/sched-12345

File: src/sample-account-setup.test.ts [hunk 3]
```diff
@@ -1,27 +1,546 @@
...
+        },
+        referenceRange: [
+          {
+            high: {
+              value: 7.0,
+            },
+          },
+        ],
+      },
+      `Observation/hemoglobinA1c`
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:113]: This also needs to be related to patient.  I'm not sure the best way to handle this, but will ask for guidance from @codyebberson and @rahul1 - namely this should be something like `Observation\patient.id+hemoglobinA1c

File: src/sample-account-setup.test.ts [hunk 4]
```diff
@@ -1,27 +1,546 @@
...
+          start: date.toISOString(),
+        },
+        category: [
+          {
+            text: 'Routine antenatal care',
+          }
+        ],
+        subject,
+      },
+      'CarePlan/active-care-plan'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:149]: Same issue. This also needs to be related to patient. I'm not sure the best way to handle this, but will ask for guidance from @codyebberson and @rahul1 - namely this should be something like `CarePlan\patient.id+antenatal.  Also, @rahul1 - we need to upgrade this to something more in line with our recommendations.

File: src/sample-account-setup.test.ts [hunk 5]
```diff
@@ -1,27 +1,546 @@
...
+          end: date.toISOString(),
+        },
+        category: [
+          {
+            text: 'Respiratory therapy',
+          }
+        ],
+        subject,
+      },
+      'CarePlan/completed-care-plan'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:183]: Same issue.

File: src/sample-account-setup.test.ts [hunk 6]
```diff
@@ -1,27 +1,546 @@
...
+        },
+        subject,
+        result: [
+          {
+            reference: createReference(hemoglobinA1c).reference,
+            display: getDisplayString(hemoglobinA1c),
+          }
+        ]
+      },
+      'DiagnosticReport/diagnostic-report'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:205]: Same issue again

File: src/sample-account-setup.test.ts [hunk 7]
```diff
@@ -1,27 +1,546 @@
...
+              }
+            }
+          },
+        ],
+        authoredOn: date.toISOString(),
+        medicationCodeableConcept: {
+          text: '72 HR Fentanyl 0.025 MG/HR Transdermal System',
+        },
+      },
+      'MedicationRequest/123'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:239]: Same issue.

File: src/sample-account-setup.test.ts [hunk 8]
```diff
@@ -1,27 +1,546 @@
...
+              }
+            }
+          },
+        ],
+        authoredOn: date.toISOString(),
+        medicationCodeableConcept: {
+          text: 'Acetaminophen 325 MG / Oxycodone Hydrochloride 10 MG Oral Tablet [Percocet]',
+        },
+      },
+      'MedicationRequest/stopped-medication'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:272]: same issue

File: src/sample-account-setup.test.ts [hunk 9]
```diff
@@ -1,27 +1,546 @@
...
+        patient: subject,
+        location: {
+          display: 'FOOMEDICAL HOSPITAL AND MEDICAL CENTERS'
+        },
+        occurrenceDateTime: date.toISOString(),
+        vaccineCode: {
+          text: 'SARS-COV-2 (COVID-19) vaccine, mRNA, spike protein, LNP, preservative free, 100 mcg/0.5mL dose',
+        },
+      },
+      'Immunization/completed-immunization'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:290]: Same issue

File: src/sample-account-setup.test.ts [hunk 10]
```diff
@@ -1,27 +1,546 @@
...
+        location: {
+          display: 'FOOMEDICAL HOSPITAL AND MEDICAL CENTERS'
+        },
+        vaccineCode: {
+          text: 'Influenza, seasonal, injectable, preservative free',
+        },
       },
-    ],
-  });
+      'Immunization/not-done-immunization'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:306]: Same issue.

File: src/sample-account-setup.test.ts [hunk 11]
```diff
@@ -1,27 +1,546 @@
...
+        valueQuantity: {
+          code: 'Cel',
+          system: 'http://unitsofmeasure.org',
+          unit: 'Cel',
+          value: 36.6,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/body-temperature'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:398]: same issue.

File: src/sample-account-setup.test.ts [hunk 12]
```diff
@@ -1,27 +1,546 @@
...
+        valueQuantity: {
+          code: 'cm',
+          system: 'http://unitsofmeasure.org',
+          unit: 'cm',
+          value: 175,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/height'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:425]: same issue.

File: src/sample-account-setup.test.ts [hunk 13]
```diff
@@ -1,27 +1,546 @@
...
+        valueQuantity: {
+          code: '/min',
+          system: 'http://unitsofmeasure.org',
+          unit: '/min',
+          value: 15,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/respiratory-rate'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:452]: Same issue.

File: src/sample-account-setup.test.ts [hunk 14]
```diff
@@ -1,27 +1,546 @@
...
+        valueQuantity: {
+          code: '/min',
+          system: 'http://unitsofmeasure.org',
+          unit: '/min',
+          value: 80,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/heart-rate'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:479]: same issue.

File: src/sample-account-setup.test.ts [hunk 15]
```diff
@@ -1,27 +1,546 @@
...
+        valueQuantity: {
+          code: 'kg',
+          system: 'http://unitsofmeasure.org',
+          unit: 'kg',
+          value: 70,
+        },
+        effectiveDateTime: date.toISOString(),
+        status: 'final',
+      },
+      'Observation/weight'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:506]: same issue.

File: src/sample-account-setup.test.ts [hunk 16]
```diff
@@ -1,27 +1,546 @@
...
+        id: 'welcome-message',
+        subject,
+        sender: subject,
+        payload: [
+          {
+            contentString: 'Hello and welcome to our practice',
+          },
+        ],
+      },
+      'Communication/welcome-message'
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:522]: Same issue

File: src/sample-account-setup.test.ts [hunk 17]
```diff
@@ -1,27 +1,546 @@
...
+    notDoneImmunization,
+    bloodPressure,
+    bodyTemperature,
+    height,
+    respiratoryRate,
+    heartRate,
+    weight,
+    message
+  };
+}
```
Comment (reshmakh) [inline src/sample-account-setup.test.ts:546]: @codyebberson and @rahul1 - this part is interesting.  Not sure the right path here, but this seems sub-optimal

File: src/sample-account-setup.ts [hunk 18]
```diff
@@ -1,102 +1,546 @@
 import { BotEvent, MedplumClient, createReference, getDisplayString } from '@medplum/core';
```
Comment (reshmakh) [inline src/sample-account-setup.ts:1]: This file will need to be changed when script is updaed

File: src/sample-account-setup.test.ts (hunk 1 above)
Comment (codyebberson) [reply src/sample-account-setup.test.ts:51]: Is the goal to have one global practitioner?  Or one practitioner per patient?

File: src/sample-account-setup.test.ts (hunk 2 above)
Comment (codyebberson) [reply src/sample-account-setup.test.ts:69]: The syntax here needs to be a search string, not by id.  You could search by identifier, such as `Schedule?identifier=xyz`

**Pull request merged**